### PR TITLE
chore: v1.0.0-beta.121

### PR DESCRIPTION
**Pull request by IgorKarpiuk**
## What/Why/How?

v1.0.0-beta.121 release:

 - join: add custom output file option(https://github.com/Redocly/redocly-cli/pull/987)
 - add an option to include webhooks to operation-4xx-response rule(https://github.com/Redocly/redocly-cli/pull/990)
 - fix: ignore case when inferring file extension from codeSample lang prop(https://github.com/Redocly/redocly-cli/pull/991)
 - chore: rename features.openapi to theme.openapi for build docs (https://github.com/Redocly/redocly-cli/pull/892)
 - chore: rename features.openapi and features.mockServer to theme (https://github.com/Redocly/redocly-cli/pull/995)
 - fix: push destination parsing (#890) 
 -  feat: new decorator info-override (https://github.com/Redocly/redocly-cli/pull/999)
 -  fix: support / separator in split command (https://github.com/Redocly/redocly-cli/pull/997)
 
## Check yourself

- [ ] Code is linted
- [ ] Tested with redoc/reference-docs/workflows
- [ ] All new/updated code is covered with tests

## Security

- [ ] Security impact of change has been considered
- [ ] Code follows company security practices and guidelines

### CODE REVIEW

File: docs/changelog.md [hunk 1]
```diff
@@ -7,6 +7,15 @@ toc:
...
 
+### Features
+
+- Added custom output file option to join command
+- Added an option to include webhooks to operation-4xx-response rule
+
+### Fixes
+
+- Ignored case when inferring file extension from codeSample lang property
+
```
Comment (adamaltman) [inline docs/changelog.md:18]:
```suggestion
### Features

- Added custom output file option to the `join` command.
- Added an option to include webhooks to [operation-4xx-response](./rules/operations-4xx-response.md) rule.

### Fixes

- Ignored case when inferring file extension from code sample `lang` property.

```

Comment (adamaltman) [reply docs/changelog.md]: Write in sentences with a full stop.

File: docs/changelog.md [hunk 2]
```diff
@@ -7,6 +7,15 @@ toc:
 
 ## 1.0.0-beta.121 (2023-01-25)
 
+### Features
+
+- Added custom output file option to the `join` command.
+- Added an option to include webhooks to [operation-4xx-response](./rules/operations-4xx-response.md) rule.
```
Comment (RomanHotsiy) [inline docs/changelog.md:13]:
```suggestion
- Added an option to include webhooks to [operation-4xx-response](./rules/operations-4xx-response.md) rule.
- Added a new built-in decorator [info-override](./decorators/info-override.md).
```

**Review by RomanHotsiy — APPROVED** [review]

File: docs/changelog.md [hunk 3]
```diff
@@ -7,6 +7,16 @@ toc:
...
+- Added a new built-in decorator [info-override](./decorators/info-override.md).
```
Comment (adamaltman) [inline docs/changelog.md:14]:
```suggestion
- Added a new built-in decorator [info-override](./decorators/info-override.md).
- Added support for `/` as a separator which puts paths into subdirectories for each path segment with the [split command](./commands/split.md).
```

**Pull request merged**